backend/pkg/server/errors.go

```diff
@@ -0,0 +1,16 @@
+package server
```

Comment:
Just out of ignorance/curiosity, is this a convention for custom error handling in Go?

Reply:
I have no idea!